core/interfaces/i_variable_map.ts

```diff
@@ -0,0 +1,66 @@
+import {IVariableModel} from './i_variable_model.js';
```

Comment:
Copyright statement plox. Here and in the other file.

Reply:
Done!